kotlin-vercel-ncc/src/jsMain/kotlin/vercel/ncc.kt

```diff
@@ -0,0 +1,12 @@
+package vercel
+
+import kotlinx.coroutines.await
+import kotlin.js.Promise
+
+@JsModule("@vercel/ncc")
+@JsNonModule
+@JsName("ncc")
+external fun nccAsync(input: String, options: NccOptions = definedExternally): Promise<NccResult>
+
+suspend fun ncc(input: String, options: NccOptions = undefined.unsafeCast<Nothing>()) =
```

Comment:
`@JsAsync`?

Reply:
Sure, but would you mind telling me what it does?
I don't even find a mention of it in seskar's readme.

Reply:
If you mark `suspend x()` with `@JsAsync` - it will call `xAsync().await()`, when you will call `x()`

Reply:
In your case function body, which you wrote for `ncc` is what plugin will generate ;)